contracts/test/consensus_gateway/SpyConsensusGateway.sol

```diff
@@ -15,10 +15,28 @@ pragma solidity ^0.5.0;
 // limitations under the License.
 
 import "../../proxies/MasterCopyNonUpgradable.sol";
+import "../../consensus-gateway/ConsensusGatewayI.sol";
 
-contract SpyConsensusGateway is MasterCopyNonUpgradable {
+contract SpyConsensusGateway is MasterCopyNonUpgradable, ConsensusGatewayI {
+
+    address public spyCore;
+    uint256 public spyFeeGasPrice;
+    uint256 public spyFeeGasLimit;
 
     function setup() public {
         // do nothing
```

Comment:
If this is not being used in any tests, add assert here to break.

Reply:
It's being used when Axiom.deployMetachainProxies is called.
`ConsensusGateway` setup data is passed as argument.